pkgs/by-name/mu/multipass/multipassd.nix

```diff
@@ -1,62 +1,48 @@
 {
+  meta,
+  multipass_src,
+  version,
+
   cmake,
   dnsmasq,
   fetchFromGitHub,
   git,
   gtest,
   iproute2,
   iptables,
+  kdePackages,
   lib,
   libapparmor,
   libvirt,
   libxml2,
-  nixosTests,
   openssl,
   OVMF,
   pkg-config,
-  qemu,
   poco,
   protobuf,
   qemu-utils,
-  qtbase,
-  qtwayland,
-  wrapQtAppsHook,
+  qemu,
   slang,
   stdenv,
   xterm,
 }:
 
 let
-  pname = "multipass";
-  version = "1.14.1";
-
   # This is done here because a CMakeLists.txt from one of it's submodules tries
   # to modify a file, so we grab the source for the submodule here, copy it into
   # the source of the Multipass project which allows the modification to happen.
   grpc_src = fetchFromGitHub {
-    owner = "CanonicalLtd";
+    owner = "canonical";
     repo = "grpc";
-    rev = "e3acf245";
-    hash = "sha256-tDc2iGxIV68Yi4RL8ES4yglJNlu8yH6FlpVvZoWjoXk=";
+    rev = "ba8e7f72a57b9e0b25783a4d3cea58c79379f194";
+    hash = "sha256-DS1UNLCUdbipn5w4p2aVa8LgHHhdJiAfzfEdIXNO69o=";
     fetchSubmodules = true;
   };
 in
 stdenv.mkDerivation {
-  inherit pname version;
-
-  src = fetchFromGitHub {
-    owner = "canonical";
-    repo = "multipass";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-i6SKiV4jwiBURx85m3u7km1dhi+fRdVpMBanlZo0VK4=";
-    fetchSubmodules = true;
-    leaveDotGit = true;
-    postFetch = ''
-      # Workaround for https://github.com/NixOS/nixpkgs/issues/8567
-      cd $out
-      rm -rf .git
-    '';
-  };
+  inherit version meta;
```

Comment:
See above

Reply:
Should be fine like this, afaik meta.position only changes when there's an explicit attribute, so only if that meta attrset has it would it change.

Reply:
Fixed, thanks :)